pkg/prom/diagnostics.go

```diff
@@ -78,13 +78,6 @@ var diagnosticDefinitions map[string]*diagnosticDefinition = map[string]*diagnos
 		Label:       "Kubecost metrics available",
 		Description: "Determine if metrics from Kubecost are available during last 5 minutes.",
 	},
-	NodeExporterDiagnosticMetricID: {
-		ID:          NodeExporterDiagnosticMetricID,
```

Comment:
Looks like you still need to delete references to this

https://github.com/opencost/opencost/blob/05f1603bc324d6ba5cd756ae123d54f10b5c56c1/pkg/prom/diagnostics.go#L29-L30

Reply:
there are references to this in kcm. I will remove have to remove from there Before I can remove it from here. Let me do that

Reply:
https://github.com/kubecost/kubecost-cost-model/pull/2998